src/modules/tests/test-data/test-data.reducer.ts

```diff
@@ -312,6 +323,39 @@ export function testDataReducer(
           },
         },
       };
+    case testDataActions.SHOW_ME_QUESTION_DANGEROUS_FAULT:
+      return {
+        ...state,
+        vehicleChecks: {
+          ...state.vehicleChecks,
+          showMeQuestion: {
+            ...state.vehicleChecks.showMeQuestion,
+            outcome: CompetencyOutcome.D,
+          },
+        },
+      };
+    case testDataActions.SHOW_ME_QUESTION_DRIVING_FAULT:
+      return {
+        ...state,
+        vehicleChecks: {
+          ...state.vehicleChecks,
+          showMeQuestion: {
+            ...state.vehicleChecks.showMeQuestion,
+            outcome: CompetencyOutcome.DF,
+          },
+        },
+      };
+    case testDataActions.SHOW_ME_QUESTION_REMOVE_FAULT:
+      return {
+        ...state,
+        vehicleChecks: {
+          ...state.vehicleChecks,
+          showMeQuestion: {
+            ...state.vehicleChecks.showMeQuestion,
+            outcome: undefined,
```

Comment:
Might need to check with @ali-campbell but I think we should be removing the outcome when it is removed..

Reply:
Okay, I am going to remove it

Reply:
Yes, if there's no need to know that a fault was removed (e.g. for audit) then we should remove the outcome.  Saves on storage and easier for validation.